www/src/pages/guidelines/design-tokens.js

```diff
@@ -209,9 +241,13 @@ const DesignTokens = ({ location }) => (
             components along the z-axis.
           </p>
         </CopyColumn>
-        <ContentColumn fullWidth width="100%">
+        <ContentColumn overflowXMobile fullWidth>
```

Comment:
@fk I could have used `overflowX="auto"` and override the css in `<ContentColumn/>`, but we only needed this css rule on mobile(below md), so I had to explicitly add overflowXMobile props and check it inside `<ContentColumn/>` component.

Reply:
👍